### ISSUE TITLE
Feedback: Week 1 Finatra Blog

### ISSUE THREAD

**Issue by shekhargulati** [shekhargulati/52-technologies-in-2016#1]
Please provide your feedback by adding a new comment

**Comment by zzztimbo** [comment]
Could you write a similar blog post for Finch?

**Comment by shekhargulati** [comment]
@zzztimbo Yes I might cover it. I am planning to cover various Twitter's Scala projects. Finch looks like a good topic to learn and write.

**Comment by dozaza** [comment]
@shekhargulati Hi, Could you please tell me some differences between finatra and other frameworks, such as play?

**Comment by fzakaria** [comment]
I would love a 'prod ready' article on Finatra incuding setting up Zipkin

**Comment by kdkocev** [comment]
Awesome article! Thank you, man!

**Comment by munendrasn** [comment]
@shekhargulati , Awesome blog.
I'm new to Finatra, your blog is really helping me in getting familiar with Finatra.
I searched internet to learn about integrating/rendering html files for Finatra Apps.
I couldn't get any good resource.
Do you have any resources for the same??
Could share those resource links here??

Thank you in advance

**Comment by kulinux** [comment]
Thank you!!.
Takes me a little bit have all configured in intellij but now works.
